Mathlib/Analysis/Convex/EGauge.lean

```diff
@@ -177,27 +182,28 @@ lemma div_le_egauge_ball (r : ℝ≥0) (x : E) : ‖x‖₊ / r ≤ egauge 𝕜
 lemma le_egauge_ball_one (x : E) : ‖x‖₊ ≤ egauge 𝕜 (ball 0 1) x := by
   simpa using div_le_egauge_ball 𝕜 1 x
 
-end SeminormedAddCommGroup
-
-section SeminormedAddCommGroup
-
-variable {𝕜 : Type*} [NormedField 𝕜] {E : Type*}
-    [NormedAddCommGroup E] [NormedSpace 𝕜 E] {c : 𝕜} {x : E} {r : ℝ≥0}
+variable {𝕜}
+variable {c : 𝕜} {x : E} {r : ℝ≥0}
 
-lemma egauge_ball_le_of_one_lt_norm (hc : 1 < ‖c‖) (h₀ : r ≠ 0 ∨ x ≠ 0) :
+lemma egauge_ball_le_of_one_lt_norm (hc : 1 < ‖c‖) (h₀ : r ≠ 0 ∨ ‖x‖ ≠ 0) :
```

Comment:
Apparently I falsely concluded that something was wrong with `egauge`; the fact that you made this work withotu changing its definition is great!